### DISCUSSION TITLE
Cannot identify peaks with find markers function

### DISCUSSION THREAD

**Discussion by jamrute** [GreenleafLab/ArchR#1717] [Questions / Documentation]
Thank you for making this package.

I have generated Multiome data, annotated nuclei from RNA information, and called peaks with MACS2 on these clusters. When I use `getMarkerFeatures ` to find marker peaks across clusters I get very few peaks identified and I am unsure why this is the case. I have attached what my heatmap looks like as well as the peak call. Only one population has a lot of marker peaks while others have almost none. I have also attached the ATAC UMAP with the RNA cluster annotations.

Thank you for your help.

[Peak-Call-Summary.pdf](https://github.com/GreenleafLab/ArchR/files/9924862/Peak-Call-Summary.pdf)
[Peak-Marker_noMax-Heatmap.pdf](https://github.com/GreenleafLab/ArchR/files/9924863/Peak-Marker_noMax-Heatmap.pdf)
[Plot-UMAP_ATAC_HARMONY-Sample-Clusters_HAR-celltypeRNA-HAR.pdf](https://github.com/GreenleafLab/ArchR/files/9924864/Plot-UMAP_ATAC_HARMONY-Sample-Clusters_HAR-celltypeRNA-HAR.pdf)

**Comment by rcorces** [comment]
I cant really know for certain what is going on. My hunch would be that some of your groups are poisoning your analysis because they have very few cells and unreliable signal. Maybe try limiting your analysis to the cell types where you have >2000 cells (using `subsetArchRProject()`) and seeing if that helps.

As an aside, we are trying to overhaul marker feature and differential feature identification but that is still in the works.

**Reply by jamrute** [reply]
Thank you for the response. I tried running the analysis on only 3 populations: SMC (65325 nuclei), myeloid (12511), and endothelium (17331) and only the myeloid population has a lot of marker peaks while the SMCs have very few to almost none. I have attached the marker heat map. When I look at QC metrics/FRiP across cell populations it is also very comparable. Here is the command I am using:

`markersPeaks <- getMarkerFeatures(
    ArchRProj = proj2, 
    useMatrix = "PeakMatrix", 
    groupBy = "celltypeRNA",
    bias = c("TSSEnrichment", "log10(nFrags)"),
    testMethod = "wilcoxon",
    useGroups = c("SMC", "Myeloid", "Fibroblast")
)
`

Thank you!

[Peak-Marker_noMax-Heatmap1.pdf](https://github.com/GreenleafLab/ArchR/files/9931495/Peak-Marker_noMax-Heatmap1.pdf)

**Reply by jamrute** [reply]
I was able to get much more marker peaks after I changed the number of cells sampled - given the SMCs had 62k nuclei sampling 500 cells may not have been enough - does this seem reasonable? Thank you!

**Reply by Lulu0015** [reply]
Hi,
I am having the same problems here and adjust the max number of cells being sampled didn't give more marker peaks. I am wondering if you have other fix towards this issue?
Thanks!
